web/src/components/tasks/RaffleBingoSteps.ts

```diff
@@ -0,0 +1,43 @@
+import { fetchTaskByFilename } from "@/lib/async-content-fetchers/fetchTaskByFilename";
+import { Task } from "@/lib/types/types";
+import { arrayOfTaskAgencies } from "@businessnjgovnavigator/shared/taskAgency";
+
+type RaffleBingoStepLabels = "Get an ID Number" | "From Your NJ Municipality";
+
+export const RaffleBingoSteps: {
+  stepLabel: RaffleBingoStepLabels;
+  fileName: string;
+  agencyId: string;
+}[] = [
+  {
+    stepLabel: "Get an ID Number",
+    fileName: "raffle-license-step-1",
+    agencyId: "games-of-chance-control-commission",
+  },
+  { stepLabel: "From Your NJ Municipality", fileName: "raffle-license-step-2", agencyId: "nj-municipality" },
+];
+
+export const getRaffleBingoTask = async (stepIndex: number): Promise<Task> => {
+  return await fetchTaskByFilename(RaffleBingoSteps[stepIndex].fileName);
+};
+
+export const getAgencyName = (agencyId: string): string => {
+  const issuingAgency = arrayOfTaskAgencies.find((agency) => agency.id === agencyId);
+  return issuingAgency!.name;
+};
```

Comment:
[boulder] We have a function `LookupTaskAgencyById` that's available in `taskAgency.ts` which does exactly this. Opt for that function over re-writing the same logic.

I understand the thought here. In the future, if you're adding something that you think would be beneficial elsewhere in the application, it could be an opportunity to write a helper function.